controlplane/kubeadm/internal/etcd/util/set.go

```diff
@@ -148,7 +148,7 @@ func (s UInt64Set) IsSuperset(s2 UInt64Set) bool {
 	return true
 }
 
-// Equal returns true if and only if s1 is equal (as a set) to s2.
+// Equals returns true if and only if s1 is equal (as a set) to s2.
```

Comment:
This looks like an accidental change
```suggestion
// Equal returns true if and only if s1 is equal (as a set) to s2.
```